charts/images.go

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package charts
+
+import _ "embed"
+
+// ImagesYAML contains the content of the images.yaml file
+//go:embed images.yaml
```

Comment:
Is it possible to move this file under `pkg/imagevector` and to have `//go:embed ../../charts/images.yaml`? It is a little bit strange to have `.go` file under `charts/`.

Reply:
I think it's not possible to use relative paths with `go:embed`, unfortunately.

From https://golang.org/pkg/embed/#hdr-Directives:

> The patterns are interpreted relative to the package directory containing the source file. The path separator is a forward slash, even on Windows systems. Patterns may not contain ‘.’ or ‘..’ or empty path elements, nor may they begin or end with a slash.

Reply:
Ah, ok, thanks for explaining.